DuckDuckGo/Assets.xcassets/IntiveLockSuccess.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "Intive-Lock-Success-96.pdf",
```

Comment:
Should "Intive" be "Invite"?  This typo is present apparently in several file names, but I'm adding a single note here to avoid overcluttering the PR with comments.